lib/beaker-rspec/helpers/serverspec.rb

```diff
@@ -95,7 +95,7 @@ def get_windows_cmd(meth, *args)
       method += "_#{subaction}" if subaction
 
       common_class = Specinfra::Command
-      base_class = common_class.const_get(:Base)
+      common_class.const_get(:Base)
```

Comment:
I think this is just unused.
```suggestion
```

I struggle to see why it can't be simplified down to
```ruby
version_class = Specinfra::Command::Windows::Base
command_class = version_class.const_get(resource_type.to_camel_case)
```

Then I dug deeper and wondered if the method was used at all. I don't think it is so I opened https://github.com/voxpupuli/beaker-rspec/pull/143 instead.